src/lib/core/ssl.c

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
I couldn't understand from EE cmake - how does it work that both OS and EE have `src/lib/core/ssl.c` but EE ignores the OS file?

Reply:
The source file is added conditionally by the OS cmake:

https://github.com/tarantool/tarantool/blob/36bf0cb36077fefbee7fd638a398d78dadad2e69/src/lib/core/CMakeLists.txt#L40-L44

`SSL_SOURCES` is defined by the EE cmake.

Reply:
Nice, thanks for the clarification.